package.json

```diff
@@ -102,14 +101,13 @@
     "fast-check": "^2.17.0",
     "jest": "^27.5.1",
     "license-checker": "^25.0.1",
+    "nlfurniss-typedoc-plugin-sourcefile-url": "^2.0.0",
```

Comment:
This movement makes sense.. I'd probably manually added it as a dependency instead of `npm install --save-dev`